app/templates/views/notifications/notification.html

```diff
@@ -35,7 +35,7 @@
       {% elif created_by %}
         by {{ created_by.name }}
       {% endif %}
-      {{ created_at|format_datetime_human }}
+      {{ created_at|format_datetime_human }} UTC
```

Comment:
Would it be more consistent to add "UTC" directly into `format_datetime_human`?

Reply:
That would probably help ensure it's there in all places that need it... 😁 

Let's add this to a cleanup PR as well, because my sense of this is it won't be quite a straightforward change - I suspect there might be a few spots where we'll introduce it and it shouldn't be there, we'll see it show up twice, or some other formatting will have to change.